src/main/java/com/minecolonies/coremod/colony/requestsystem/management/handlers/ResolverHandler.java

```diff
@@ -147,7 +148,7 @@ public static void removeResolver(final IStandardRequestManager manager, final I
             throw new IllegalArgumentException("The given resolver and the resolver that is registered with its token are not the same.");
         }
 
-        if (manager.getResolverRequestMap().containsKey(registeredResolver.getRequesterId()) && manager.getResolverRequestMap().get(registeredResolver.getRequesterId()).size() > 0)
+        if (manager.getRequestResolverRequestAssignmentDataStore().getAssignments().containsKey(registeredResolver.getRequesterId()) && manager.getRequestResolverRequestAssignmentDataStore().getAssignments().get(registeredResolver.getRequesterId()).size() > 0)
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Split this 260 characters long line (which is greater than 180 authorized). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS00103)